src/line_opacity.jl

```diff
@@ -76,7 +85,9 @@ function line_profile(temp::F, atomic_mass::F, ξ::F, line::NamedTuple, wls::Abs
 
     #Voigt function parameters
     v = @. abs(λs-λ0) / Δλ_D
-    a = @. λs^2/(4π * c_cgs) * γ / Δλ_D
+    #technically, a depends on λ, but at the sub-percent level, which is below the precision of our
+    #voigt approximation
+    a = λ0^2/(4π * c_cgs) * γ / Δλ_D
```

Comment:
Do you think it would potentially make sense to document somewhere that this change right here places a constraint on `line.wl` and `window_size`? This just seems like an unexpected place that the code could break if the rest of the codebase ever gets extended for UV spectra (Maybe you could add an `assert` statement at the start of `line_absorption` with a comment explaining why it's there).

I realize that this is future proofing. If you don't think it's a concern, feel free to ignore this

Reply:
This is a good idea.  I can add a comment to the source, but should we have a central place where we record the limitations of various approximations?  I definitely don't want to have an in-code check here.

Alternatively, this is a very small performance gain, maybe I should just revert it?  I'm kind of leaning towards that.

Reply:
I went ahead and rolled back that approximation.